dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-email/src/test/java/org/apache/dolphinscheduler/plugin/alert/email/MailUtilsTest.java

```diff
@@ -162,4 +162,14 @@ public void testTableAttachmentFile() {
         AlertResult alertResult = mailSender.sendMails("gaojing", content);
         Assertions.assertFalse(alertResult.isSuccess());
     }
+
+    @Test
+    public void testReceiversWithInvalidAddress() {
+        String content = list2String();
+        emailConfig.put(AlertConstants.NAME_SHOW_TYPE, ShowType.TABLE_ATTACHMENT.getDescp());
+        emailConfig.put(MailParamsConstants.NAME_PLUGIN_DEFAULT_EMAIL_RECEIVERS, "347801120@qq.com,xxx@qq.com");
+        mailSender = new MailSender(emailConfig);
+        AlertResult alertResult = mailSender.sendMails("gaojing", content);
+        Assertions.assertFalse(alertResult.isSuccess());
+    }
```

Comment:
For the bug you fix, you should test the input of, say, 3 email addresses, 2 of which are valid and 1 of which is invalid.